lib/turbo_boost/commands/runner.rb

```diff
@@ -241,6 +245,28 @@ def response_type
     :unknown
   end
 
+  # Indicates if a TurboStream template exists for the current action.
+  # Any template with the format of :turbo_boost or :turbo_stream format is considered a match.
```

Comment:
turbo boost templates are new, right?

Reply:
They've been supported for a while now given the Rails mechanics around mime types etc.

Reply:
thats 🤯 . needs an example or README entry, I guess. I wasn't aware of it.